### ISSUE TITLE
Multi line 'li' is not indenting properly

### ISSUE THREAD

**Issue by manikantag** [AmruthPillai/Reactive-Resume#12]
This style is causing the multi-line `<li/>` to have not proper indentation (show below):

```
ul {
    list-style: disc;
    list-style-position: inside!important;  -------> causing the second line to not have any indent
}
ul {
    list-style: none;
    margin: 0;
    padding: 0; ----> need to use some padding when removing `list-style-position`
}
```

![image](https://user-images.githubusercontent.com/317299/77824721-cbcd0e80-712a-11ea-9994-eea66ee24588.png)

**Comment by AmruthPillai** [comment]
This issue has been resolved in #15, thank you for the bug report :)

**Comment by manikantag** [comment]
Thanks for the quick fix
